modules/calcite/src/test/java/org/apache/ignite/internal/processors/query/calcite/planner/AbstractPlannerTest.java

```diff
@@ -674,13 +674,17 @@ protected static IgniteSchema createSchema(TestTable... tbls) {
         }
 
         /** */
-        TestTable(String name, RelDataType type, RewindabilityTrait rewindable, double rowCnt) {
+        TestTable(String name, RelDataType type, RewindabilityTrait rewindable, Double rowCnt) {
             protoType = RelDataTypeImpl.proto(type);
             this.rewindable = rewindable;
-            this.rowCnt = rowCnt;
             this.name = name;
 
             desc = new TestTableDescriptor(this::distribution, type);
+            stat = (rowCnt == null) ? null : new ObjectStatisticsImpl(rowCnt.longValue(), Collections.emptyMap());
```

Comment:
stat not declaired as Nullable

Reply:
why you didn`t initialize _stat_ if rowCnt == null ?

Reply:
removed